src/delibird/database/schema.py

```diff
@@ -89,7 +114,7 @@ def create_arrow_schema(engine, dsn, table_name):
     for col in cursor.description:
         if engine == "postgresql":
             type_list.append(pa.field(col.name, arrow_type(engine, col)))
-        elif engine == "oracle":
+        elif engine == "oracle" or engine == "mysql":
```

Comment:
oracle and mysql cursor description is same ?

Reply:
Yes! the col description of Oracle and Mysql are all tuples.